ui/static/ui/js/main.js

```diff
@@ -1,39 +1,41 @@
-define(['jquery', 'bootstrap', 'icheck'], function($) {
-    jQuery(document).ready(function($){
-      $('input.icheck-11').iCheck({
-          checkboxClass: 'icheckbox_square-blue',
-          radioClass: 'iradio_square-blue'
-      });
-      $("[data-toggle=popover]").popover();
+define(['jquery', 'bootstrap', 'icheck'], function(jQuery) {
```

Comment:
I investigated and it looks like bootstrap is used in the lateral panels so it needs to be required here